src/Thumbnail/FormatThumbnail.php

```diff
@@ -29,6 +35,33 @@ public function __construct($defaultFormat)
         $this->defaultFormat = $defaultFormat;
     }
 
+    /**
+     * @param string           $id
+     * @param ResizerInterface $resizer
+     */
+    public function addResizer($id, ResizerInterface $resizer)
+    {
+        if (!isset($this->resizers[$id])) {
+            $this->resizers[$id] = $resizer;
+        }
+    }
+
+    /**
+     * @param string $id
+     *
+     * @throws \Exception
+     *
+     * @return ResizerInterface
+     */
+    public function getResizer($id)
+    {
+        if (!isset($this->resizers[$id])) {
+            throw new \Exception('Resizer with id: '.$id.' is not attached.');
```

Comment:
Please quote the id:
```suggestion
            throw new \LogicException(sprintf('Resizer with id: "%s" is not attached.', $id));
```

Reply:
Please add a unit test for this

Reply:
And please use sprintf for concatenation